main.py

```diff
@@ -161,7 +161,8 @@ def reports():
 @app.route('/reports/<report_id>')
 def report(report_id):
     report = report_util.get_report(report_id)
-    if not report:
+    external_reports = report_util.get_external_reports()
+    if not report or report_id in external_reports:
```

Comment:
Could we 302 redirect to the external report URL instead? That would also provide a more memorable URL for the dashboard.

Reply:
Good idea.

Why 302 (temporary) over 301 (permanent) though? Or you think it might eventually be hosted on httparchive.org?

Implemented 301 for now in https://github.com/HTTPArchive/httparchive.org/pull/441/commits/1c3e6c5678774099827753dfe9eaed0dea05f7f9 but easy to update if you want.

Reply:
We want to be able to change the URL if needed. My understanding of 301 was that it could cause the old URL to get stuck in some caches.

Reply:
Fair enough. Not too concerned about difference to be honest as don't want this page indexed from here, so as you say, safer to go 302. Changed!

Happy to release now?

Reply:
Yes, LGTM. Thanks!